shared/constants/fs.tsx

```diff
@@ -89,7 +89,11 @@ export const tlfSyncDisabled: Types.TlfSyncDisabled = {
   mode: Types.TlfSyncMode.Disabled,
 }
 
-export const makeTlfSyncPartial = ({enabledPaths}: Partial<Types.TlfSyncPartial>): Types.TlfSyncPartial => ({
+export const makeTlfSyncPartial = ({
```

Comment:
i did this as an experiment, if you're just pulling out certain fields, its faster to just pull them out